src/components/PluginSettings/components/SettingSelectComponent.tsx

```diff
@@ -44,7 +46,8 @@ export function SettingSelectComponent({ option, pluginSettings, definedSettings
 
     return (
         <Forms.FormSection>
-            <Forms.FormTitle>{option.description}</Forms.FormTitle>
+            <Forms.FormTitle>{wordsToTitle(wordsFromCamel(id))}</Forms.FormTitle>
+            <Forms.FormText className={Margins.bottom20} type="description">{option.description}</Forms.FormText>
```

Comment:
20 seems too much, discord seems to use 16! also perhaps we could match their font size

![image](https://github.com/user-attachments/assets/1807fcd0-9422-401a-a2a5-793f7a3f1c10)
![image](https://github.com/user-attachments/assets/dc3a35ba-1b7d-4caf-b29d-a059d1ada2ff)


```suggestion
            <Forms.FormText className={Margins.bottom16} type="description">{option.description}</Forms.FormText>
```